src/main/java/de/chojo/repbot/commands/Thankwords.java

```diff
@@ -11,31 +14,55 @@
 import de.chojo.repbot.data.GuildData;
 import net.dv8tion.jda.api.Permission;
 import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
 
 import javax.sql.DataSource;
+import java.io.IOException;
 import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
 import java.util.regex.Pattern;
 import java.util.regex.PatternSyntaxException;
 import java.util.stream.Collectors;
 
+import static org.slf4j.LoggerFactory.getLogger;
+
 public class Thankwords extends SimpleCommand {
 
     private final GuildData data;
     private final Localizer loc;
+    private final ThankwordsContainer thankwordsContainer;
+
+    private static final Logger log = getLogger(Thankwords.class);
 
     public Thankwords(DataSource dataSource, Localizer localizer) {
-        super("thankwords", new String[] {"tw"},
+        super("thankwords", new String[]{"tw"},
                 "command.thankwords.description",
                 null,
                 subCommandBuilder()
                         .add("add", "<pattern>", "command.thankwords.sub.add")
                         .add("remove", "<pattern>", "command.thankwords.sub.remove")
                         .add("list", null, "command.thankwords.sub.list")
                         .add("check", "<Sentence>", "command.thankwords.sub.check")
+                        .add("loadDefault", "[language]", "command.thankwords.sub.loadDefault")
                         .build(),
                 Permission.MANAGE_SERVER);
         data = new GuildData(dataSource);
         loc = localizer;
+        ThankwordsContainer thankwordsContainer;
+        try {
+            thankwordsContainer = new ObjectMapper()
```

Comment:
Do you really want to do IO in the constructor?

Reply:
Yeah thought that as well.
Alternatives would be a static block or static factory method.
What do you think?

Reply:
factory method:

```java
ps ThankWords of(DataSource dataSource, Localizer localizer)
```